client/signup/main.jsx

```diff
@@ -183,13 +188,13 @@ const Signup = React.createClass( {
 	componentDidMount() {
 		debug( 'Signup component mounted' );
 		SignupProgressStore.on( 'change', this.loadProgressFromStore );
-		SignupProgressStore.on( 'change', this.loadDependenciesFromStore );
+		this.sigunpListener = SignupDependencyStore.reduxStore.subscribe( this.loadDependenciesFromStore );
```

Comment:
Also, from the Redux documentation, the property's name should reflect that it contains:

> A function that unsubscribes the change listener.

For instance, `this.unsubscribe = SignupDep….reduxStore.subscribe( … )`